### ISSUE TITLE
rename KERNEL_* fields in inparam_advanced

### ISSUE THREAD

**Issue by martinvandriel** [geodynamics/axisem#41]
Given that most AxiSEM users outside the developers use AxiSEM to compute Instaseis databases and have not heard about kernels, we might want to rename these fields. What about INSTASEIS_ * instead, as all Kernel users know Instaseis, but not the other way around?

**Comment by tnissen** [comment]
Agreed, but we should keep in mind that over short or long the kernel 
capability will be added too, and we should avoid yet another renaming 
then. Maybe something generic, such as WAVEFIELDS_* ?

On 11/23/15 13:04, Martin van Driel wrote:
>
> Given that most AxiSEM users outside the developers use AxiSEM to 
> compute Instaseis databases and have not heard about kernels, we might 
> want to rename these fields. What about INSTASEIS_ * instead, as all 
> Kernel users know Instaseis, but not the other way around?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/geodynamics/axisem/issues/41>.
>

-- 
Tarje

<>--<>--<>--<>--<>--<>
Dept. of Earth Sciences
Oxford University
South Parks Road
Oxford OX1 3AN; UK
tel: +44 1865 282149
web: seis.earth.ox.ac.uk
<>--<>--<>--<>--<>--<>
